applications/app/controllers/WebAppController.scala

```diff
@@ -18,4 +30,15 @@ object WebAppController extends Controller with ExecutionContexts {
   def manifest() = Action {
     Cached(3600) { Ok(templates.js.webAppManifest()) }
   }
+
+  def offlinePage() = Action.async { implicit request =>
+    if (conf.Switches.OfflinePageSwitch.isSwitchedOn) {
+      withCrossword("quick", 14127) { (crossword, content) =>
```

Comment:
Yup, still figuring out how to make that work with service workers. Not sure yet…